.github/workflows/ci.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     - name: Install wasm-pack
       uses: taiki-e/cache-cargo-install-action@v1
       with:
-        tool: wasm-pack@0.12.1
```

Comment:
we should bump it also in release-plz workflow, just check if it still works with next template

Reply:
Interestingly in release-plz we install it via wasm-pack-action, and since we don't set version there it automatically downloads the latest one. 
I'll open a separate PR where we align the two, since I assume we want to have consistency there :P